### ISSUE TITLE
Definition: Known Good Set (KGS)

### ISSUE THREAD

**Issue by guettli** [pypa/python-packaging-user-guide#234]
https://packaging.python.org/en/latest/glossary/#term-known-good-set-kgs

> A set of distributions at specified versions  ...

There is not definition of "distribution".

There are definitions for concrete distributions:

 * Binary Distribution
 * Built Distribution
 * Distribution Package

Please be explicit: What kind of "distributions" is meant in the above definition of KGS?

**Comment by petri** [comment]
I am not an authority by any means on this, but as far as I know, a KGS can refer to different kinds of distributions, and it perfectly ok to do so. Thus the term is distribution kind agnostic, is it not?

**Comment by pradyunsg** [comment]
Distribution Package is defined.

There is at least one place in the document (in Built Distribution definition) that links "distribution" to "Distribution Package" heading. Maybe, this use should as well.

Alternatively, mention that distribution packages may be simply called distributions?

**Comment by guettli** [comment]
@pradyunsg you mean that the word "distribution" in the definition of "Known Good Set (KGS)" means "Distribution Package"?

Yes, maybe. Maybe not. I don't know.

**Comment by pradyunsg** [comment]
> you mean that the word "distribution" in the definition of "Known Good Set (KGS)" means "Distribution Package"?

Yes. That's what it looks like to me.

**Comment by pradyunsg** [comment]
Actually, reading this again, it might make more sense to rephrase this to:

"A set of releases of different projects which are known to be compatible with each other."
